indico/modules/events/templates/header.html

```diff
@@ -38,15 +38,20 @@
        title="{% trans %}Change theme{% endtrans %}"></a>
     <ul class="dropdown">
         {% set endpoint = 'timetable.timetable' if event.type == 'conference' else 'events.display' %}
-        {% for id, data in themes.items()|sort(attribute='1.name')
-           if event.can_manage(session.user) or data.user_visible %}
-            <li>
-                {% if id == theme %}
-                    <a class="disabled"><strong>{{ data.name }}</strong></a>
-                {% else %}
-                    <a href="{{ url_for(endpoint, event, view=id) }}">{{ data.name }}</a>
-                {% endif %}
-            </li>
+        {% for user_visible, themes in themes.items()|sort(attribute='1.name')|groupby('1.user_visible')|reverse
+           if event.can_manage(session.user) or user_visible %}
```

Comment:
It was already like this before, but I would reverse the order of the checks. Checking the truthiness of a simple variable is usually much cheaper than any method call